README.md

```diff
@@ -120,6 +120,30 @@ and tactics specialized for monadic programs (see
 
 A tutorial for the Lean backend is available [here](./tests/lean/Tutorial.lean).
 
+## Quick start for Nix users
+
+Assuming Nix is installed, with a support for Flakes (`*`):
+
+```console
+$ nix run github:aeneasverif/aeneas#charon -L
```

Comment:
Could you add a comment like "run charon with the exact version required by aeneas" above?

Reply:
Yep, can do that.

Reply:
But wait, running charon like this will run it with the version of aeneas, no?

Reply:
Yes but this may not be obvious, hence why I suggest adding a comment that explains what's happening